src/environment.cpp

```diff
@@ -2180,39 +2180,29 @@ void ServerEnvironment::deactivateFarObjects(bool force_delete)
 
 			if(block)
 			{
-				if (block->m_static_objects.m_stored.size() >= g_settings->getU16("max_objects_per_block")) {
-					warningstream << "ServerEnv: Trying to store id = " << obj->getId()
-							<< " statically but block " << PP(blockpos)
-							<< " already contains "
-							<< block->m_static_objects.m_stored.size()
-							<< " objects."
-							<< " Forcing delete." << std::endl;
-					force_delete = true;
-				} else {
-					// If static counterpart already exists in target block,
-					// remove it first.
-					// This shouldn't happen because the object is removed from
-					// the previous block before this according to
-					// obj->m_static_block, but happens rarely for some unknown
-					// reason. Unsuccessful attempts have been made to find
-					// said reason.
-					if(id && block->m_static_objects.m_active.find(id) != block->m_static_objects.m_active.end()){
-						warningstream<<"ServerEnv: Performing hack #83274"
-								<<std::endl;
-						block->m_static_objects.remove(id);
-					}
-					// Store static data
-					u16 store_id = pending_delete ? id : 0;
-					block->m_static_objects.insert(store_id, s_obj);
+				// If static counterpart already exists in target block,
+				// remove it first.
+				// This shouldn't happen because the object is removed from
+				// the previous block before this according to
+				// obj->m_static_block, but happens rarely for some unknown
+				// reason. Unsuccessful attempts have been made to find
+				// said reason.
+				if(id && block->m_static_objects.m_active.find(id) != block->m_static_objects.m_active.end()){
```

Comment:
While I didn't actually edit that line aside from removing an indentation level, I suppose I can fix it...
What is the space issue, and where would you prefer I split the line? I'm not terribly familiar with MT's style guidelines.